### PR TITLE
Py3 list copy

### PR DESCRIPTION
**Pull request by alexwenbj**

### CODE REVIEW

**Comment by alexwenbj** [comment]
@s-cork The list copy() function was added in Python 3.3, it is not in Python 2.But there was an error in my previous implementation: the list copy is shallow copy.I modified it in this version.

**Comment by rixner** [comment]
I feel like this is not something that should have it's own flag.  I think moving forward we had agreed to use the ```python3``` flag instead of adding new flags (like ```list_copy```).  I'd favor removing ```list_clear``` as well, and just using ```python3``` for both of those features.

Otherwise, this looks pretty straightforward.

**Comment by alexwenbj** [comment]
@rixner What you mean is just like the below in copy() function:
```
if(Sk.__future__.python3){
    //copy implementation
}
```
and remove the list_copy config in env.js?

File: src/env.js [hunk 1]
```diff
@@ -118,6 +120,7 @@ Sk.configure = function (options) {
     Sk.bool_check(Sk.__future__.dunder_next, "Sk.__future__.dunder_next");
     Sk.bool_check(Sk.__future__.dunder_round, "Sk.__future__.dunder_round");
     Sk.bool_check(Sk.__future__.list_clear, "Sk.__future__.list_clear");
```
Comment (albertjan) [inline src/env.js]: And this:

```suggestion
```

File: src/env.js [hunk 2]
```diff
@@ -118,6 +120,7 @@ Sk.configure = function (options) {
...
+    Sk.bool_check(Sk.__future__.list_copy, "Sk.__future__.list_copy");
```
Comment (albertjan) [inline src/env.js]: And this: 
```suggestion
    Sk.bool_check(Sk.__future__.list_copy, "Sk.__future__.list_copy");
```

File: src/env.js [hunk 3]
```diff
@@ -205,6 +208,7 @@ Sk.configure = function (options) {
     Sk.switch_version("round$", Sk.__future__.dunder_round);
     Sk.switch_version("next$", Sk.__future__.dunder_next);
     Sk.switch_version("clear$", Sk.__future__.list_clear);
```
Comment (albertjan) [inline src/env.js]: Change this to use `python3`
```suggestion
    Sk.switch_version("clear$", Sk.__future__.python3);
```

File: src/env.js [hunk 4]
```diff
@@ -205,6 +208,7 @@ Sk.configure = function (options) {
...
+    Sk.switch_version("copy$", Sk.__future__.list_copy);
```
Comment (albertjan) [inline src/env.js]: And do the same here:

```suggestion
    Sk.switch_version("copy$", Sk.__future__.python3);
```

**Comment by alexwenbj** [comment]
@albertjan Got it & updated it.

**Comment by rixner** [comment]
@alexwenbj this looks good now!  Sorry for not being more clear about my requested changes.

**Comment by alexwenbj** [comment]
@rixner Thank you.

**Pull request merged**